bff/values.yaml

```diff
@@ -63,6 +74,10 @@ appConfig:
   REDIS_PORT: 6379
   INVITATION_URL_LOGIN: 
 
+appConfigSidecar:
+  REDIS_HOST: redis-master.redis
+  REDIS_PORT: 6379
```

Comment:
I think we'd want `HOST` and `PORT` as well right?

Reply:
@gregmccoy I thought of adding it during the deployment to avoid creating empty default values